values/roundtriptest/int_test.go

```diff
@@ -41,19 +49,22 @@ func TestPositiveIntDictionaryEncodeAndDecode(t *testing.T) {
 	testEncodeAndDecodeInt(t, data, meta, vals)
 }
 
-func TestNegativeIntDictionaryEncodeAndDecode(t *testing.T) {
+func TestNegativeIntRawSizeEncodeAndDecode(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
 	meta := values.IntValuesMetadata{
 		Size: 512,
-		Min:  -1,
-		Max:  -1,
+		Max:  0,
+		Min:  mockDataOutlierMin,
```

Comment:
Same question, I assume this is used to determine we should use varint encoding?

Reply:
Added a comment above the const def.